src/component/CardCVCInput/CardCVCInput.stories.tsx

```
@@ -0,0 +1,30 @@
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    cardCVC: 123,
+    onChange: () => {},
+    hasError: false,
+  },
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)